.jenkinsci/github-api.groovy

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env groovy
+
+def mergePullRequest() {
```

Comment:
Pls, add some description of methods because now it is not so obvious what really this method does.

Reply:
@muratovv added